cmd/statreceiver/luacfg/cfg.go

```diff
@@ -60,6 +60,7 @@ func (scope *Scope) register(name string, val interface{}, pusher func(l *lua.St
 // Run runs the Lua source represented by in
 func (scope *Scope) Run(in io.Reader) error {
 	l := lua.NewState()
+	lua.OpenLibraries(l)
```

Comment:
Odd, you did need to change the code to make this work?

Reply:
Yes, I talked to jt about it, the osenv library was disabled